inference-engine/src/mkldnn_plugin/emitters/jit_load_store_emitters.cpp

```diff
@@ -857,7 +857,7 @@ template <typename Vmm>
                     }
                 }
             } else {
-                // direct mov_dw available only on avx512, emulate with pack_dw + permute + pure store
+                // direct mov_dw available only on avx512, emulate with pack_dw + transpose + pure store
```

Comment:
permute here is in fact name of the vector instruction, so not necessary to change the wording.

Reply:
Fixed